.vscode/launch.json

```diff
@@ -57,7 +63,8 @@
             "stopAtEntry": false,
             "env": {
                 "ASPNETCORE_ENVIRONMENT": "Development",
-                "DataStore": "CosmosDb"
+                "DataStore": "CosmosDb",
```

Comment:
>CosmosDb [](http://example.com/codeflow?start=30&length=8)

should it have cosmosDb host and key as well?

Reply:
updated